### ISSUE TITLE
[3.3.5][Talent] Improved Leader of the Pack

### ISSUE THREAD

**Issue by D4R4** [TrinityCore/TrinityCore#12830]
Hi,
There is a problem with this Talent: Improved Leader of the Pack "http://www.wowwiki.com/Improved_Leader_of_the_Pack"
The talent in itself has many features and they all work fine except the last section which is not yet scripted where the rest of the talent in Unit.cpp "HandleDummyAuraProc" section,

the feature that is not currently working is: ". In addition, you gain 8% of your maximum mana when you benefit from this heal."

there is a spell for this proc (68285) which I think should be casted with custom attributes like these other examples: `basepoints0 = int32(CountPctFromMaxMana(triggerAmount));` but I'm not good enough to make it work ;)

The source is the talent info itself,

tl;dr: the mana regen part of this talent is not yet implanted
core: c741d2682a183
##
